src/diffpy/morph/morphs/morphsqueeze.py

```diff
@@ -78,6 +78,13 @@ def morph(self, x_morph, y_morph, x_target, y_target):
         coeffs = [self.squeeze[f"a{i}"] for i in range(len(self.squeeze))]
         squeeze_polynomial = Polynomial(coeffs)
         x_squeezed = self.x_morph_in + squeeze_polynomial(self.x_morph_in)
+        strictly_increasing_x = (np.diff(x_squeezed) > 0).all()
+        if not strictly_increasing_x:
+            raise ValueError(
+                "Computed squeezed x is not strictly increasing. "
+                "Please change the input x_morph or the squeeze "
```

Comment:
If possible let's have a more helpfule message here.  I am a user.  OK, I need to change those things but what do I change them to?  How do I change them?